### PR TITLE
Update Websub and xslt bbes

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/websub-hub-client-sample/websub_hub_client_sample.metatags [hunk 1]
```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates the usage of the WebSub Hub Client subscribe and unsubscribe to updates for a particular topic at the hub.
```
Comment (Samallama) [inline examples/websub-hub-client-sample/websub_hub_client_sample.metatags:1]: Edit: This BBE demonstrates the usage of the WebSub Hub Client subscribe and unsubscribe to update for a particular topic at the hub.

File: examples/websub-internal-hub-sample/websub_internal_hub_sample.metatags [hunk 2]
```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates how WebSub publisher publishes updates to the internal hub and subscriber subscribes to the publisher's hub to receive notifications.
```
Comment (Samallama) [inline examples/websub-internal-hub-sample/websub_internal_hub_sample.metatags:1]: Edit: This BBE demonstrates how a WebSub publisher publishes updates to the internal hub and a subscriber subscribes to the publisher's hub to receive notifications.

File: examples/websub-remote-hub-sample/websub_remote_hub_sample.metatags [hunk 3]
```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates a simple WebSub publisher and subscriber scenario.
```
Comment (Samallama) [inline examples/websub-remote-hub-sample/websub_remote_hub_sample.metatags:1]: Edit: This BBE demonstrates a simple WebSub publisher and subscriber scenario.

File: examples/websub-service-integration-sample/websub_service_integration_sample.metatags [hunk 4]
```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates any of the Ballerina's WebSub components (Publisher, Hub, or Subscriber) could be used with non-Ballerina components.
```
Comment (Samallama) [inline examples/websub-service-integration-sample/websub_service_integration_sample.metatags:1]: Edit: This BBE demonstrates how any of Ballerina's WebSub components (Publisher, Hub, or Subscriber) could be used with non-Ballerina components.

File: examples/xslt-transformation/xslt_transformation.metatags [hunk 5]
```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates how the XML content can be transformed to HTML using a given XSL transformation.
```
Comment (Samallama) [inline examples/xslt-transformation/xslt_transformation.metatags:1]: Edit: This BBE demonstrates how the XML content can be transformed into HTML using a given XSL transformation.

File: examples/xslt-transformation/xslt_transformation.metatags [hunk 6]
```diff
@@ -0,0 +1,2 @@
...
+keywords: ballerina, ballerina by example, bbe, xslt
```
Comment (Samallama) [inline examples/xslt-transformation/xslt_transformation.metatags:2]: Shall we add xml and html as keywords too?

File: examples/websub-internal-hub-sample/publisher.bal [hunk 7]
```diff
@@ -1,4 +1,4 @@
-// The Ballerina WebSub Publisher brings up the internal Ballerina hub, registers a topic at the hub, and publishes updates to the topic.
+// The [Ballerina WebSub Publisher]() brings up the internal Ballerina hub, registers a topic at the hub, and publishes updates to the topic.
```
Comment (chamil321) [inline examples/websub-internal-hub-sample/publisher.bal:1]: Ballerina WebSub Publisher link is missing

File: examples/websub-service-integration-sample/websub_service_integration_sample.metatags [hunk 8]
```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates how any of Ballerina's WebSub components (Publisher, Hub, or Subscriber) could be used with non-Ballerina components.
+keywords: ballerina, ballerina by example, bbe, websub, hub, subscriber, publisher
```
Comment (chamil321) [inline examples/websub-service-integration-sample/websub_service_integration_sample.metatags:2]:
```suggestion
keywords: ballerina, ballerina by example, bbe, websub, hub, subscriber, publisher, integration
```

File: examples/xslt-transformation/xslt_transformation.metatags [hunk 9]
```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates how the XML content can be transformed into HTML using a given XSL transformation.
+keywords: ballerina, ballerina by example, bbe, xslt, xml, html
```
Comment (chamil321) [inline examples/xslt-transformation/xslt_transformation.metatags:2]:
```suggestion
keywords: ballerina, ballerina by example, bbe, xslt, xml, html, xsl, transformation
```

**Pull request merged**